instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -373,14 +381,19 @@ def _populate_span(
     ):
         if not span.is_recording():
             return
+
         statement = self.get_statement(cursor, args)
+        collection_name = self.get_collection_name(statement)
+
         span.set_attribute(
             SpanAttributes.DB_SYSTEM, self._db_api_integration.database_system
         )
         span.set_attribute(
             SpanAttributes.DB_NAME, self._db_api_integration.database
         )
         span.set_attribute(SpanAttributes.DB_STATEMENT, statement)
+        if collection_name:
+            span.set_attribute(DB_COLLECTION_NAME, collection_name)
```

Comment:
But as I said in the comments before, I wouldn't introduce new semantic convention attributes in this PR.